src/store/modules/groups.js

```diff
@@ -164,8 +162,9 @@ export const actions = {
 
     const userId = rootGetters['auth/userId']
     if (!group.members.includes(userId)) {
-      Toast.create.warning(i18n.t('GROUP.NONMEMBER_REDIRECT'))
-      throw new Error('user is not a member')
+      const message = i18n.t('GROUP.NONMEMBER_REDIRECT')
+      dispatch('routeError/set', message, { root: true })
```

Comment:
Better to set the translation key, not the actual translation text? (otherwise will not update if locale changes)